tests/acceptance-tests/src/acceptance/git_buildpack_test.go

```diff
@@ -0,0 +1,29 @@
+package acceptance_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+
+	"github.com/cloudfoundry-incubator/cf-test-helpers/cf"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/generator"
+)
+
+var _ = Describe("GitBuildpack", func() {
+	var (
+		appName string
+	)
+
+	It("should fail when using a buildpack from a git url", func() {
+		appName = generator.PrefixedRandomName("CATS-APP-")
+		Expect(
+			cf.Cf(
+				"push", appName, "--no-start",
+				"-m", DEFAULT_MEMORY_LIMIT,
+				"-p", "../../example-apps/static-app",
+				"-b", "https://github.com/cloudfoundry/staticfile-buildpack.git",
+				"-d", config.AppsDomain,
+			).Wait(DEFAULT_TIMEOUT)).To(Exit(1))
```

Comment:
This could give false-positive results if CF or GitHub is down.

Can we be more specific about the error that we're catching?